install/includes/config_default.php

```diff
@@ -31,6 +31,7 @@
         'log' => array(
             'archive' => '1',
             'maxsize' => '102400',
+            'filter' => E_ALL ^ E_DEPRECATED,
```

Comment:
Doesn't this say "all or deprecated", which is the same as "all"? I would expect `E_ALL & ~E_DEPRECATED` instead.

Reply:
I'm pretty sure it's a XOR, meaning E_ALL or E_DEPRECATED, but not both. I'm not strong with bitwise though, always have to look it up!

Reply:
Ah, thanks.

Reply:
The resulting integer value is the same, BTW.

Reply:
It is all "minus" deprecated. ie (a && !b) || (!a && b) on bits

> The resulting integer value is the same, BTW.

Yes because in that case, it's always (a && !b) that wins!